frontend/pipeline.yml

```diff
@@ -83,6 +83,7 @@ test:frontend:unit:
     - npm run test -- --reporter=default --reporter=junit --outputFile.junit=./junit.xml --coverage
   after_script:
     - mkdir logs && cp frontend/logs/* logs/
+  allow_failure: true
```

Comment:
I would add a comment here explicitly with the reference to JIRA. It is important that we revert this eventually.